modules/ROOT/pages/migration-patterns-reconnection-strategies.adoc

```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-From a syntax point of view, reconnection hasn't changed much between Mule 3 and Mule 4. However, there are significant behavior differneces.
+Reconnection syntax hasn't changed much between Mule runtime engine (Mule) 3 and Mule 4. However, there are significant behavior differneces.
 
 In Mule 3, reconnection strategies were specified on each connector's config element. These strategies had two purposes:
 
 * To reconnect when a running application looses connection to an endpoint
-* To validate all connections when the application is being deployed.
+* To validate all connections when the application is being deployed
 
-So, suppose an application which connects to a mysql Database. If that connection cannot be established at deployment time, the deployment will fail. This was to guarantee that all deployed apps are in a functioning state.
+So, suppose an application which connects to a mysql database. If that connection cannot be established at deployment time, the deployment will fail. This was to guarantee that all deployed apps are in a functioning state.
```

Comment:
I know this is beyond the scope of what you are doing, but this whole thing is a mess. First sentence is a fragment. Last sentence combines past with present.